### PR TITLE
Use a separate http server for the "end state" API

### PR DESCRIPTION
**Pull request by LukasGasior1**

### CODE REVIEW

File: radixdlt-core/radixdlt/src/main/java/com/radixdlt/RadixNodeModule.java [hunk 1]
```diff
@@ -137,8 +137,11 @@ public final class RadixNodeModule extends AbstractModule {
           Network.SANDPITNET.getId(), new GenericTestnetForksModule(),
           Network.LOCALNET.getId(), new GenericTestnetForksModule());
 
-  private static final int DEFAULT_CORE_PORT = 3333;
-  private static final String DEFAULT_BIND_ADDRESS = "0.0.0.0";
+  private static final int DEFAULT_PRIMARY_API_PORT = 3333;
+  private static final String DEFAULT_PRIMARY_API_BIND_ADDRESS = "127.0.0.1";
```
Comment (malexandru-rdx) [inline radixdlt-core/radixdlt/src/main/java/com/radixdlt/RadixNodeModule.java:141]: Is the default change from `0.0.0.0` to `127.0.0.1` intended?

**Review by malexandru-rdx — APPROVED** [review]
LGTM!

File: radixdlt-core/radixdlt/src/main/java/com/radixdlt/RadixNodeModule.java (hunk 1 above)
Comment (LukasGasior1) [reply radixdlt-core/radixdlt/src/main/java/com/radixdlt/RadixNodeModule.java:141]: Yes, core API really shouldn't be exposed externally (unless one knows what they're doing)

Comment (LukasGasior1) [reply radixdlt-core/radixdlt/src/main/java/com/radixdlt/RadixNodeModule.java:141]: Yes. And the other one is `RADIXDLT_END_STATE_API_BIND_ADDRESS`

**Pull request merged**